infrastructure/staging/dashboard.tf

```diff
@@ -54,18 +54,14 @@ data "aws_iam_role" "dashboard_task_role" {
   name = module.dashboard.task_role_name
 }
 
-resource "aws_iam_role_policy" "dashboard_read_wellcomecollection_storage_bucket" {
```

Comment:
Remove access from 'staging' to the production wellcome bucket - 'staging-prod' now has access to this.